demos/include/iot_demo_runner.h

```diff
@@ -39,21 +39,61 @@
 
 /* Individual demo task entry definitions */
 #if defined( CONFIG_MQTT_DEMO_ENABLED )
-    #define DEMO_entryFUNCTION    RunMqttDemo
+    #define DEMO_entryFUNCTION              RunMqttDemo
+    #if defined( democonfigMQTT_ECHO_TASK_STACK_SIZE )
```

Comment:
If ```democonfigMQTT_ECHO_TASK_STACK_SIZE``` is not defined, then whatever is the value of ```democonfigDEMO_STACKSIZE``` will be used. This can mean that the demos can still be broken in that case. Is that correct?

Reply:
democonfigMQTT_ECHO_TASK_STACK_SIZE  (or any other demo specific config) is defined in aws_demo_config.h, if the demo needs specific stack size.  Since release 1.5, code started  using single stack size, so these defines in aws_demo_config.h were ignored.  Default size does not mean demo is broken, it means default stack size will be used.